The SDL headers include the DirectFB headers, but without proper C FLAGS
they are not found. This works around the issue.
--- a/CMakeLists.txt	2023-12-06 09:43:31.514059721 +0100
+++ b/CMakeLists.txt	2023-12-06 09:48:03.708234182 +0100
@@ -85,6 +85,9 @@ if(DISABLE_SYSMIDI)
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DNOSYSMIDI")
 endif()
 
+include(FindPkgConfig)
+pkg_search_module(DIRECTFB REQUIRED directfb)
+include_directories(${DIRECTFB_INCLUDE_DIRS})
 
 target_include_directories(tfe PRIVATE TheForceEngine)
 
